src/providers/Gardens.js

```diff
@@ -6,17 +6,15 @@ import React, {
   useState,
 } from 'react'
 import { getGardens } from '@1hive/connect-gardens'
-
+import { useWallet } from './Wallet'
 import { ConnectedGardenProvider } from './ConnectedGarden'
 import { useDebounce } from '@hooks/useDebounce'
 import useGardenFilters from '@hooks/useGardenFilters'
 import { useGardenRoute } from '@hooks/useRouting'
-import { useWallet } from './Wallet'
-
-import { fetchFileContent } from '../services/github'
-import { getVoidedGardensByNetwork } from '../voided-gardens'
 import { mergeGardenMetadata } from '@utils/garden-utils'
 import { testNameFilter } from '@utils/garden-filters-utils'
+import { fetchFileContent } from '../services/github'
+import { getVoidedGardensByNetwork } from '../voided-gardens'
```

Comment:
Why did you change the import orders here?

Reply:
Humm, I think it was why i REBASE to get the new commits and on the merge it get swap.
I'll fix that. Ty for get it.